### PR TITLE
Add xz-utils requirement

### PR DESCRIPTION
**Pull request by gardouille**
##### SUMMARY
With **deb** parameter, the **apt** module try to extract the control file from the deb package to install. As "new" deb package now contains **xz** archive, the remote host need a tool to extract the wanted file.

##### ISSUE TYPE
- Docs Pull Request

##### COMPONENT NAME
apt

##### ANSIBLE VERSION
```
ansible 2.6.4
  config file = /home/gardouille/.ansible.cfg
  configured module search path = [u'/home/gardouille/.ansible/plugins/modules', u'/usr/share/ansible/plugins/modules']
  ansible python module location = /home/gardouille/debops-venv/local/lib/python2.7/site-packages/ansible
  executable location = /usr/local/bin/ansible
  python version = 2.7.15+ (default, Aug 31 2018, 11:56:52) [GCC 8.2.0]
```

##### ADDITIONAL INFORMATION
* Without **xz-utils** package, i got this issue :
```
TASK [ipr-cnrs.fusioninventory : Ensure fusioninventory-agent package from URL] ********************************************************
fatal: [REMOTE_HOST.DOMAIN]: FAILED! => {"changed": false, "msg": "Unable to install package: There is no member named 'control'"}
```
* Once the **xz-utils** package, Ansible successfully install the wanted package : 
```
TASK [ipr-cnrs.fusioninventory : Ensure fusioninventory-agent package from URL] ********************************************************
changed: [REMOTE_HOST.DOMAIN]
```

### CODE REVIEW

File: lib/ansible/modules/packaging/os/apt.py [hunk 1]
```diff
@@ -124,6 +124,7 @@
    - python-apt (python 2)
    - python3-apt (python 3)
    - aptitude (before 2.4)
+   - xz-utils
```
Comment (acozine) [inline lib/ansible/modules/packaging/os/apt.py]: If `xz-utils` is only a requirement with the `deb` parameter, please don't add it here - this section is for general requirements for the module.

File: lib/ansible/modules/packaging/os/apt.py [hunk 2]
```diff
@@ -133,6 +134,7 @@
      (If you typo C(foo) as C(fo) apt-get would install packages that have "fo" in their name with a warning and a prompt for the user.
      Since we don't have warnings and prompts before installing we disallow this.Use an explicit fnmatch pattern if you want wildcarding)
    - When used with a `loop:` each package will be processed individually, it is much more efficient to pass the list directly to the `name` option.
+   - The deb parameter require xz-utils package in order to extract the control file of the deb package to install.
```
Comment (acozine) [inline lib/ansible/modules/packaging/os/apt.py]: I'd put this line under the current line 95, in the description of the `deb` parameter.

**Review by acozine — CHANGES_REQUESTED** [review]

File: lib/ansible/modules/packaging/os/apt.py (hunk 2 above)
Comment (gardouille) [reply lib/ansible/modules/packaging/os/apt.py]: Yeah i wasn't sure for both point. Thanks.

**Comment by acozine** [comment]
Thanks @gardouille for this! We're having some CI challenges (not related to this PR) - as soon as the PR passes the test suite, I'll merge it.

**Review by acozine — APPROVED** [review]

**Pull request merged**